pkg/probes/probes.go

```diff
@@ -19,13 +19,14 @@ import (
 	"fmt"
 	"net/http"
 
-	"github.com/networkservicemesh/networkservicemesh/pkg/probes/health"
-
 	"github.com/sirupsen/logrus"
+
+	"github.com/networkservicemesh/networkservicemesh/pkg/probes/health"
 )
 
 const (
-	healthcheckProbesPort = "0.0.0.0:5555"
+	probePort             = 5555
+	healthcheckProbesPort = "0.0.0.0:" + string(probePort)
```

Comment:
string(int) will convert int to unicode symbol...

Reply:
please make probePort as string

Reply:
- [ ] use fmt.Sprintf() to build healthcheckProbesPort

Reply:
- [ ] assign probePort as string